src/FINDOLOGIC/Export/Data/SalesFrequency.php

```diff
@@ -2,12 +2,34 @@
 
 namespace FINDOLOGIC\Export\Data;
 
-use FINDOLOGIC\Export\Helpers\UsergroupAwareNumericValue;
+use FINDOLOGIC\Export\Helpers\UsergroupAwareSimpleValue;
+use FINDOLOGIC\Export\Helpers\EmptyValueNotAllowedException;
 
-class SalesFrequency extends UsergroupAwareNumericValue
+class ValueIsNotPositivIntegerException extends \RuntimeException
+{
+    public function __construct()
+    {
+        parent::__construct('Value is not a positiv integer!');
```

Comment:
The message would be more expressive if it was something like `$fieldName is not a positive integer!` by passing the name of the field as a constructor parameter.

Reply:
Done!